mlflow/utils/databricks_utils.py

```diff
@@ -161,6 +163,9 @@ def is_in_databricks_job():
     except Exception:
         return False
 
+def is_in_databricks_serving_environment():
+    return "MODEL_VERSION_ID" in os.environ
```

Comment:
Seems like a pretty generic env var...is there anything more specific we can use?